### PR TITLE
BMW ConnectedDrive: Update vehicle finder description.

### PR DESCRIPTION
**Pull request by rikroe**
## Proposed change

Add information about HA's location being sent when using BMW `vehicle_finder` service.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/60938
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/bmw_connected_drive.markdown [hunk 1]
```diff
@@ -154,6 +154,15 @@ The service `bmw_connected_drive.light_flash` flashes the lights of the vehicle.
 
 The service `bmw_connected_drive.find_vehicle` requests the vehicle to update the GPS location. This can be used for older vehicles which don't automatically send the updated GPS location. The vehicle is identified via the parameter `vin`.
 
+<div class="note">
+  Using this service will <strong>send your Home Assistant location to BMW</strong>, as this is required by the API (like sharing your mobile phone's location with the MyBMW app for vehicle tracking).
+  If you do not want this, trigger the <code>vehicle_finder</code> service from your phone and it should update in Home Assistant within 5 minutes.
```
Comment (frenck) [inline source/_integrations/bmw_connected_drive.markdown]: Please avoid using HTML instead, use Markdown.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/bmw_connected_drive.markdown (hunk 1 above)
Comment (rikroe) [reply source/_integrations/bmw_connected_drive.markdown]: Just found the documentation regarding new lines in note boxes and adjusted accordingly.

File: source/_integrations/bmw_connected_drive.markdown [hunk 2]
```diff
@@ -154,9 +154,9 @@ The service `bmw_connected_drive.light_flash` flashes the lights of the vehicle.
 
 The service `bmw_connected_drive.find_vehicle` requests the vehicle to update the GPS location. This can be used for older vehicles which don't automatically send the updated GPS location. The vehicle is identified via the parameter `vin`.
 
-<div class="note">
-  Using this service will <strong>send your Home Assistant location to BMW</strong>, as this is required by the API (like sharing your mobile phone's location with the MyBMW app for vehicle tracking).
-  If you do not want this, trigger the <code>vehicle_finder</code> service from your phone and it should update in Home Assistant within 5 minutes.
+<div class="note warning">
+  Using this service will **send your Home Assistant location to BMW**, as this is required by the API (like sharing your mobile phone's location with the MyBMW app for vehicle tracking).
+  If you do not want this, trigger the `vehicle_finder` service from your phone and it should update in Home Assistant within 5 minutes.
```
Comment (frenck) [inline source/_integrations/bmw_connected_drive.markdown]: Aah I see the issue you probably had with this. You'll need to add an empty line, to allow the Markdown parser to switch HTML/Markdown context to work.

```suggestion
<div class="note warning">

  Using this service will **send your Home Assistant location to BMW**, as this is required by the API (like sharing your mobile phone's location with the MyBMW app for vehicle tracking).
  If you do not want this, trigger the `vehicle_finder` service from your phone and it should update in Home Assistant within 5 minutes.
  
```

Comment (rikroe) [reply source/_integrations/bmw_connected_drive.markdown]: Yes, that was the problem :)
I added the additional lines in https://github.com/home-assistant/home-assistant.io/pull/20570/commits/3548453a1e4870281aa56e2b3ca354836451bddb so the current preview (https://deploy-preview-20570--home-assistant-docs.netlify.app/integrations/bmw_connected_drive/#vehicle-finder) is looking good!

**Review by frenck — APPROVED** [review]
Thanks, @rikroe 👍

**Pull request merged**